lib/modules/datasource/galaxy-collection/readme.md

```diff
@@ -0,0 +1,31 @@
+By default the `ansible-galaxy` collection will lookup your dependencies on https://galaxy.ansible.com.
```

Comment:
```suggestion
By default, the `galaxy-collection` datasource checks for dependencies on `https://galaxy.ansible.com`.
```

I think the name of the datasource is `galaxy-collection`, so we should use that.